utils/build-repo-internal.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -l
+
+set -eux
+
+REPO_PATH=$1
+REPO=$(basename "$REPO_PATH")
+
+cp -r "$REPO_PATH" .
+cd "$REPO"
+
+sudo yum install -y cmake3
+
+eval "cmake3 ."
+eval "make"
```

Comment:
Use this piece of code instead:
```
mkdir build
cd build
cmake ..
make
```

1. It's more safe to use a build directory to avoid source pollution and of course collisions.
2. eval is not a good way to execute a command. :)

Reply:
No longer used, because SRPM build commands (with cmake calls) are used instead.